### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by codingwithdavsy**
please pull and merge in my feature branch

### CODE REVIEW

**Comment by codingwithsydav** [comment]
ok i'll take a look now

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 just playing with branch protection and prs
 
 comment from sydav local repo
```
Comment (codingwithsydav) [inline README.md:4]: before i approve your pr please edit this line

**Review by codingwithsydav — CHANGES_REQUESTED** [review]
please edit the line i specified

File: README.md (hunk 1 above)
Comment (codingwithdavsy) [reply README.md]: ok i changed it as requested

Comment (codingwithsydav) [reply README.md]: very nice edit 10/10

**Pull request merged**